l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -721,8 +722,12 @@ def setDatiPagamento(self, invoice, body):
     def setAttachments(self, invoice, body):
         if invoice.fatturapa_doc_attachments:
             for doc_id in invoice.fatturapa_doc_attachments:
+                nome_attachment = doc_id.datas_fname if len(
+                    doc_id.datas_fname) <= 60 else os.path.join([
```

Comment:
Why `os.path.join` here? I don't think it gives the desired result. You can use `'.'.join`